Source/Core/Gen3/Searchers/WildSearcher3.cpp

```diff
@@ -24,9 +24,10 @@
 #include <Core/RNG/LCRNG.hpp>
 #include <Core/Util/EncounterSlot.hpp>
 
-WildSearcher3::WildSearcher3(u16 tid, u16 sid, u8 genderRatio, Method method, const StateFilter &filter) :
+WildSearcher3::WildSearcher3(u16 tid, u16 sid, u8 genderRatio, Method method, const StateFilter &filter, bool isRSEVersion) :
     WildSearcher(tid, sid, genderRatio, method, filter), cache(method), searching(false), progress(0)
```

Comment:
Same comment as generator, move line outside of constructor to `rse(rse)`

Reply:
done